pkgs/top-level/all-packages.nix

```diff
@@ -5730,6 +5730,10 @@ in
 
   parallel = callPackage ../tools/misc/parallel { };
 
+  parallel-full = callPackage ../tools/misc/parallel/wrapper.nix {
+    extraPerlPackages = with perlPackages; [ DBI DBDPg DBDSQLite DBDCSV TextCSV ];
```

Comment:
All-packages.nix is quite hard to navigate. A user looking at the package will likely look at wrapper.nix first. Therefore I would move the default perlPackages to `wrapper.nix`.